opentelemetry-sdk/src/trace/sampler/jaeger_remote/sampler.rs

```diff
@@ -203,7 +203,13 @@ impl JaegerRemoteSampler {
                     // send request
                     match Self::request_new_strategy(&client, endpoint.clone()).await {
                         Ok(remote_strategy_resp) => strategy.update(remote_strategy_resp),
-                        Err(err_msg) => global::handle_error(TraceError::Other(err_msg.into())),
+                        Err(err_msg) => {
+                            otel_warn!(
+                                name: "JaegerRemoteSampler.UpdateStrategy.RequestFailed",
+                                message = "Failed to fetch the new sampling strategy from remote endpoint. This may cause the sampler to use stale configuration until the next successful update.",
```

Comment:
If this is the failure in the first time fetching, there is no stale configuration.

Reply:
The extra msg is newly added? Probably no needed.

Reply:
> The extra msg is newly added? Probably no needed.

This is the updated extra message based on the comments. Why is this not required?
``
message = "Failed to fetch the sampling strategy from the remote endpoint. The last successfully fetched configuration will be used if available; otherwise, the default sampler will be applied until a successful configuration fetch.",
``

Reply:
> > The extra msg is newly added? Probably no needed.
> 
> This is the updated extra message based on the comments. Why is this not required? `message = "Failed to fetch the sampling strategy from the remote endpoint. The last successfully fetched configuration will be used if available; otherwise, the default sampler will be applied until a successful configuration fetch.",`

This looks very nice!